datafusion/common/src/column.rs

```diff
@@ -127,6 +127,13 @@ impl Column {
             })
     }
 
+    /// return the column's name.
```

Comment:
I added some small access APIs to make the example easier to read. I can revert these or put them in a different PR if reviewers prefer